pkg/k8shandler/reconciler.go

```diff
@@ -130,7 +129,14 @@ func ReconcileForKibanaSecret(requestClient client.Client) (err error) {
 	}
 
 	// call for Kibana to restart itself (e.g. delete its pods)
-	return clusterLoggingRequest.RestartKibana()
+	if err := clusterLoggingRequest.removeKibana(); err != nil {
```

Comment:
as part of removing this logic from CLO we won't need `ReconcileForKibanaSecret` anymore. that logic should reside in EO

Reply:
Moved to elasticsearch-operator